OutOfSchool/OutOfSchool.DataAccess/Repository/NotificationRepository.cs

```diff
@@ -47,10 +47,8 @@ public async Task<IEnumerable<Notification>> SetReadDateTimeByType(string userId
     /// <inheritdoc/>
     public async Task SetReadDateTimeForAllUnreaded(string userId, DateTimeOffset dateTime)
     {
-        // TODO: replace it in EFCore8 with bulk update
-        await db.Database.ExecuteSqlRawAsync(
-            @"UPDATE Notifications SET ReadDateTime = {0} WHERE UserId = {1} AND ReadDateTime IS NULL;",
-            dateTime,
-            userId);
+        await db.Notifications
+            .Where(n => n.UserId == userId && n.ReadDateTime == null)
+            .ExecuteUpdateAsync(s => s.SetProperty(n => n.ReadDateTime, dateTime));
```

Comment:
(Irrelevant code changes) Thanks for the improvement, but It is usually better to introduce such changes in separate branch and PR.

Reply:
Additionally - this change reduces test coverage and introduces "single await in return" smell (I don't know if any related term exists) when you have method like `async Method() => await otherMethod()`, that works, but adds extra context switches, and degrades performance

Reply:
@VadymLevkovskyi should i make this method synchronous?

Reply:
It depends on your meaning of "synchronous" ;) 

As for me, method that returns `Task` is still asynchronous, even when it doesn't have `async`/`await` keywords.
So next version is still asynchronous, but doesn't introduce state machine and context switching as `async`/`await `version does
```
    public Task SetReadDateTimeForAllUnreaded(string userId, DateTimeOffset dateTime)
    {
        return db.Notifications...

```

Reply:
Fixed. Thank you!